### PR TITLE
clevis metadata maintenance

### PR DESCRIPTION
**Pull request by ZelenyMartin**

### CODE REVIEW

File: tang/Sanity/simple-tang-server-start/main.fmf [hunk 1]
```diff
@@ -1,24 +1,24 @@
 summary: Run tang server, which stays running after test finishes. For manual testing
     with --reserve parameter.
-description: ''
-contact: Martin Zelený <mzeleny@redhat.com>
+component+:
```
Comment (sarroutbi) [inline tang/Sanity/simple-tang-server-start/main.fmf:3]: Is this "component+" a new feature, or is it a typo? If not a typo ... what is it used for?

Comment (ZelenyMartin) [reply tang/Sanity/simple-tang-server-start/main.fmf:3]: This is the feature. It extends the inherited list.

Comment (sarroutbi) [reply tang/Sanity/simple-tang-server-start/main.fmf:3]: OK. Sorry for my ignorance. Thanks for clarifying

**Review by sarroutbi — COMMENTED** [review]
Changes LGTM. Please, consider I am noob regarding fmf, so I would recommend approval from @sopos

**Review by sopos — APPROVED** [review]
Ok with me. I would consider also removing NoRHEL* tags as it does not have any effect anymore.

**Pull request merged**